include/cooperative_perception/angle.hpp

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2022 Leidos
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ANGLE_HPP
+#define ANGLE_HPP
```

Comment:
The include guards should have the project name to avoid collisions with other projects. Consider the alternative
```cpp
COOPERATIVE_PERCEPTION_ANGLE_HPP
```
This is the convention for the other header files.